ethers-contract/src/multicall/mod.rs

```diff
@@ -100,7 +98,7 @@ pub type Result<T, M> = std::result::Result<T, MulticallError<M>>;
 /// Helper struct for managing calls to be made to the `function` in smart contract `target`
 /// with `data`.
 #[derive(Clone, Debug)]
-pub struct Call {
+pub struct GenericCall {
```

Comment:
what's the reason behind renaming this?

can we not keep it as `Call`, because this is breaking, right?

Reply:
I think it's a bug with rust analyzer, there's multiple Call types even if they are aliased

Reply:
Found in the tg chat. I can rename it back if you want @mattsse

Reply:
let's avoid breaking unless we think that it's a basically unused api